code/log/lib/Loot/Log/Internal.hs

```diff
@@ -40,7 +46,7 @@ import qualified Data.Text as T
 
 
 -- | An event that gets logged (in most cases just text).
-newtype LogEvent = LogEvent { getLogEvent :: Text }
```

Comment:
Was it unnecessary? All newtypes should have `unX` internal method if it can be useful at least once.

Reply:
I decided between exporting this and removing it (to avoid warnings). It looks like fully module- internal stuff, so I chose the second option.